src/registrar/tests/test_models.py

```diff
@@ -63,6 +64,31 @@ def test_full_create(self):
         application.other_contacts.add(contact)
         application.save()
 
+    def test_domain_info(self):
+        """Can create domain info with all fields."""
+        user, _ = User.objects.get_or_create()
+        contact = Contact.objects.create()
+        domain, _ = Domain.objects.get_or_create(name="igorville.gov")
+        information = DomainInformation.objects.create(
+            creator=user,
+            organization_type=DomainInformation.OrganizationChoices.FEDERAL,
+            federal_type=DomainInformation.BranchChoices.EXECUTIVE,
+            is_election_board=False,
+            organization_name="Test",
+            address_line1="100 Main St.",
+            address_line2="APT 1A",
+            state_territory="CA",
+            zipcode="12345-6789",
+            authorizing_official=contact,
+            submitter=contact,
+            purpose="Igorville rules!",
+            anything_else="All of Igorville loves the dotgov program.",
+            is_policy_acknowledged=True,
+            domain=domain,
+        )
+        information.other_contacts.add(contact)
+        information.save()
```

Comment:
**suggestion**: Can we test at least one of the properties of the `information` object after we save? Also a check on the reverse lookup of the domain.domain_info
```suggestion
        information.save()
        self.assertEqual(information.domain.id, domain.id)
        self.assertEqual(information.id, domain.domain_info.id)
```